gotham_reqwest/src/main.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2020 O.S. Systems Sofware LTDA
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use std::sync::{Arc, Mutex};
+#[macro_use]
+extern crate gotham_derive;
+
+use rwcst::prelude::*;
```

Comment:
Please add `use derive_more::From` here and avoid the full path derive down the code.